src/test/resources/restores/caseclaim.xml

```diff
@@ -225,7 +225,7 @@
         <create>
             <case_type>child_case</case_type>
             <case_name>The Dark Knight</case_name>
-            <owner_id>4bc2b147d2e03808b69351c6373defa0</owner_id>
+            <owner_id>47ef5ea3cc348fb6538702449c855dcd</owner_id>
```

Comment:
This test was getiing purged earlier after restores causing a few tests to fail. I made the `owner_id` here same as the user id in this restore to avoid this getting purged after restores.